executor/partition_table_test.go

```diff
@@ -3924,6 +3924,7 @@ func TestPartitionOnMissing(t *testing.T) {
 		id INT NOT NULL,
 		listid INT,
 		name varchar(10)
+		//primary key (listid) clustered
```

Comment:
`//` is not a SQL comment, please use `--` or `#` instead if you want to comment out a part of the SQL text.